tests/unit/test_models.py

```diff
@@ -6,7 +6,6 @@
 import pytest
 
 import pybop
-from examples.standalone.model import ExponentialDecay
```

Comment:
Let's keep some tests on the standalone model in `tests/unit/test_standalone.py`